src/main/webapp/app/admin/standardized-competencies/standardized-competency-management.component.ts

```diff
@@ -85,6 +86,11 @@ export class StandardizedCompetencyManagementComponent extends StandardizedCompe
         this.dialogErrorSource.unsubscribe();
     }
 
+    exportStandardizedCompetencyCatalog() {
+        const url = `/api/admin/standardized-competencies/export`;
```

Comment:
I'd leave it here since it will not be used anywhere else and it keeps the code of the export function in one place. 
If this ever changes it can still be refactored then :)